usort/util.py

```diff
@@ -28,26 +28,23 @@ def timed(msg: str) -> Generator[None, None, None]:
     TIMINGS.append((msg, after - before))
 
 
-def print_timings(fn: Callable[[str], None] = print) -> None:
+def get_timings() -> Sequence[Tuple[str, float]]:
+    try:
+        return list(TIMINGS)
+    finally:
+        TIMINGS.clear()
+
+
+def print_timings(
+    fn: Callable[[str], None] = print, timings: Sequence[Timing] = ()
```

Comment:
I understand it's after an arg that has a default, but I feel like a default on `timings` doesn't make much sense.